codegen/gradle.properties

```diff
@@ -1,2 +1,3 @@
 smithyVersion=1.28.1
-smithyGradleVersion=0.6.0
\ No newline at end of file
+smithyGradleVersion=0.6.0
+org.gradle.jvmargs=-Xmx1024m
```

Comment:
Was this change intended to be included in the PR? It isn't called out in the description and seems unrelated.